types/stpadoptions.pp

```diff
@@ -0,0 +1,15 @@
+# Matches Pad Options Keys And Values
+type Etherpad::Stpadoptions = Struct[
```

Comment:
What does the `St` stand for? Should this just be `Padoptions`?

Reply:
Fixed.